bokeh/models/plots.py

```diff
@@ -27,10 +27,12 @@
     Any,
     List as TList,
     overload,
+    Union,
 )
 
 # External imports
 from typing_extensions import Literal
+import xyzservices
```

Comment:
FYI our single source of truth for runtime dependencies is here:

https://github.com/bokeh/bokeh/blob/a06c4b7d71130f265abbe814831960dfcff0e063/_setup_support.py#L45-L53